### DISCUSSION TITLE
What do I need to change to make my website look more like another person's?

### DISCUSSION THREAD

**Discussion by Some-random** [alshedivat/al-folio#2676] [Q&A]
This might be asking for a lot, but since I'm not very familiar with website building, any help would be greatly appreciated.

Here’s the [source code](https://github.com/Some-random/Some-random.github.io) for my website, and here’s my [website](https://www.jiangdongwei.com/). Additionally, you can find another person's website [source code](https://github.com/teacherpeterpan/teacherpeterpan.github.io) here, and his [website](https://liangmingpan.bio/) here. I'd like to make my website look more like his.

Specifically, I need help with:

1  Placing the profile picture on the same line as my name.
2  Adding a dividing line between sections, like the one between the news and selected publications on his site.

**Comment by george-gca** [comment]
About the profile picture, you'll have to do a little tinkering yourself, but this is all about html/css positioning.

To add lines between the sections you can do this in markdown using `---` or in html with `<hr>` tag.
